### PR TITLE
Break out node config into stand-alone playbook

### PR DESCRIPTION
**Pull request by mtnbikenc**
Added byo/openshift-node/config.yml as a new entry point for performing node installation.  This playbook must be run after masters have already been deployed.

Excluder disable/enable for both nodes and masters has been moved their respective config.yml. This created a common pattern that could be used for both installation and scaleup playbooks.

### CODE REVIEW

File: playbooks/byo/openshift-node/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+- include: ../openshift-cluster/initialize_groups.yml
+
+- include: ../../common/openshift-cluster/std_include.yml
+
+- include: ../../common/openshift-node/config.yml
+  vars:
+    openshift_cluster_id: "{{ cluster_id | default('default') }}"
+    openshift_debug_level: "{{ debug_level | default(2) }}"
+    openshift_master_etcd_hosts: "{{ groups.etcd | default([]) }}"
+    openshift_master_etcd_port: 2379
```
Comment (abutcher) [inline playbooks/byo/openshift-node/config.yml]: I wonder if we need all these vars? I suspect we could remove `openshift_cluster_id` at least.

Comment (mtnbikenc) [reply playbooks/byo/openshift-node/config.yml]: I was thinking the same.  I think most of that was related to the `bin/cluster` stuff.  I'm going to drop the vars and test an install.

**Comment by mtnbikenc** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
[merge]

**Comment by mtnbikenc** [comment]
Flake https://github.com/openshift/origin/issues/16005
[merge]

**Pull request merged**